src/System.ServiceModel.Syndication/src/App10Constants.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+namespace Microsoft.ServiceModel.Syndication
+{
+    internal static class App10Constants
```

Comment:
To be consistent with the other components in corefx, the source files should be nested under their namespace components (for example, see [`System.Collections`' layout](https://github.com/dotnet/corefx/tree/master/src/System.Collections/src/System/Collections/Generic)).

Instead of:

`src/System.ServiceModel.Syndication/src/App10Constants.cs`

Change to:

`src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/App10Constants.cs`

(applies throughout)

Reply:
Yes, will change this later in separate PR.